packages/foreman/rubygem-multi_json/rubygem-multi_json.spec

```diff
@@ -5,7 +5,7 @@
 Summary: A gem to provide swappable JSON backends
 Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 1.12.2
```

Comment:
the tfm-ror52 SCL contained version 13.x so this still won't obsolete it.

Reply:
Yepp, that's still to be fixed.